.ado/templates/react-native-macos-init.yml

```diff
@@ -63,7 +63,7 @@ steps:
   - task: CmdLine@2
     displayName: Bump package version
     inputs:
-      script: node .ado/bumpFileVersions.js
+      script: node scripts/bump-oss-version.js --nightly
```

Comment:
I dont think this will work on the stable branch. I think you need the same two versions of this as you have in the publish workflow (one for master, one for stable branches)

Reply:
This doesn’t actually publish, so are you thinking that different test runs might be using the same commit hash?

Reply:
yeah, maybe its ok.
It might be better if the PR was testing the version bump that will happen during publish, rather than the one that happens only on master.  But We can look at if thats needed later. 
Let's get this PR in, since it strictly makes things better.